src/notation/view/widgets/editstaff.ui

```diff
@@ -1076,7 +1105,6 @@
   <tabstop>showTimesig</tabstop>
   <tabstop>showBarlines</tabstop>
   <tabstop>hideSystemBarLine</tabstop>
-  <tabstop>mergeMatchingRests</tabstop>
```

Comment:
This particular deletion should be undone to preserve keyboard accessibility

Reply:
On macOS I can't get it to tab to any of the comboboxes but perhaps that is a platform issue. (There is a macOS keyboard option to get it to tab to  all controls, but I don't know if these settings control it.) Anyway, I'll change it.